wandb/sdk/wandb_init.py

```diff
@@ -449,12 +449,67 @@ def init(
     id=None,
     settings: Union[Settings, Dict[str, Any], None] = None,
 ) -> RunBase:
-    """Initialize a wandb Run.
-
+    """Initialize W&B
+    If called from within Jupyter, initializes a new run and waits for a call to
+    `wandb.log` to begin pushing metrics.  Otherwise, spawns a new process
+    to communicate with W&B.
     Args:
-        entity: alias for team.
-        team: personal user or team to use for Run.
-        project: project name for the Run.
+        job_type (str, optional): The type of job running, defaults to 'train'
+        dir (str, optional): An absolute path to a directory where metadata will
+            be stored.
+        config (dict, argparse, or tf.FLAGS, optional): The config parameters
```

Comment:
I think `tf.FLAGS` has been deprecated in favor of `absl.flags`

Reply:
ok